### DISCUSSION TITLE
Setting up events when using in Vanilla JS

### DISCUSSION THREAD

**Discussion by mob343** [simeydotme/svelte-range-slider-pips#137] [Q&A] [answered]
I'm using Svelte on a HTML page using vanilla javascript.

How do I setup an 'onChange' event listener when doing this?

This is my code:

`  var MySlider = new RangeSliderPips({
    target: document.querySelector("#capacity-slider"),
    props: { 
      values: [minSize,maxSize],
      min: minSize,
      max: maxSize,
      range: true,
      pips: true,
      float: true
    },    
  });`

I suppose I just need the JS equivalent of this:

`on:change={(e) => { console.log(e.detail) }}`

Thanks in advance for any help!

**Comment by mob343** [comment, marked as answer]
Nevermind, I figured it out!

`var MySlider = new RangeSliderPips({
    target: document.querySelector("#capacity-slider"),
    props: { 
      values: [minSize,maxSize],
      min: minSize,
      max: maxSize,
      range: true,
      pips: false,
      float: true
    },    
  }).$on( 'change', (e) => {    
    let handle = e.detail.activeHandle;
    let value = e.detail.value;
    }
  });
`

**Comment by simeydotme** [comment]
Glad you sorted it! :D
It's great to see people using this as a vanilla module! Happy!
